docs/integrations/databases/open-telemetry/redis.md

```diff
@@ -0,0 +1,135 @@
+id: redis
+title: Sumo Logic Open Telemetry App for Redis
+sidebar_label: Redis
+description: Documentation for Open Telemetry App for Redis.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/databases/redis.png')} alt="Thumbnail icon" width="50"/>
+
+[Redis](https://redis.io/docs/about/) is an in-memory data structure implementing a distributed, in-memory key-value database with optional durability.
+
+The Sumo Logic App for Redis helps you monitor the Redis database cluster. The preconfigured dashboards provide detailed analysis based on logs. The panels provide details such as RDBMemory Usage, events, RDB and AOF events.
+
+Redis logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/schematics-redis.png')} alt="Redis Schematics" />
+
+##Redis Log Types
```

Comment:
The heading is not appearing properly when I render it. I thing the problem is since there is no space between ## and the actual text.